### ISSUE TITLE
ciox:Krankenhausverwaltung skos:closeMatch bb:HospitalAdministrationDepartment v2018-10-22

### ISSUE THREAD

**Issue by AlfredWinter** [IMISE/snik-ontology#180]
The edge "ciox:Krankenhausverwaltung skos:closeMatch bb:HospitalAdministrationDepartment" is incorrect.

**Details**
Das eine ist ein entity type, das andere eine Funktion! Das lässt sich nicht so einfach korrigieren. Ich habe Birgit gebeten, dass mit Frau Jahn zu diskutieren und dann Anna Lörke darauf anzusetzen. 

Was ist hier zu tun?
Abteilungen aus dem Blauen Buch (star um Hospital Administration Department) sind als entity types modelliert. Das müssten aber Roles sein. Dann müssen die hier angesprochene Beziehung auch eine zwischen Role und Funktion werden. 
Aber es muss bei jeder Abteilung mit star geschaut werden, was drumrum ist und dann muss ggf. angepasst werden. Aufwändig!

**Comment by KonradHoeffner** [comment]
Kann auch Herr Wermke übernehmen, habe ihn aber noch nicht im Repository.

**Comment by LudwigWermke** [comment]
@AnnaLoerke hast du hier schon was gemacht/ vorbereitet? 
LG
Ludwig

**Comment by AnnaLoerke** [comment]
Eine Vorarbeit von Birgit und mir liegt hier schon im Büro (227 im Regal; Ausdruck von SNIK mit Klebezetteln, falls du gucken möchtest). Allerdings sind einige Unklarheiten aufgetreten, sodass wir Rücksprache mit Prof. Winter halten wollen.

**Comment by AlfredWinter** [comment]
Bitte mit BS und FJ klären

**Comment by KonradHoeffner** [comment]
Haben wir den Zettel noch? In der nächsten Jour Fixe ansehen.
Anfangen mit Hospital Administration Department, angleichen mit CIOX Krankenhausverwaltung.

**Comment by KonradHoeffner** [comment]
Ich kann Annas Zettel leider nirgendwo finden und hoffe, den haben wir nicht entsorgt als sie gegangen ist. @ThomasPause: Kannst du dich daran erinnern, haben wir vielleicht ein Foto davon?

**Comment by BirgitSNIK** [comment]
Um welche Zettel geht es denn?
Anna hat ja auch viele ihrer Grafen in ihre Bachelorarbeit integrierte (Anhang). Vielleicht hilft euch das? 
Birgit
 
Von: Konrad Höffner <notifications@github.com> 
Gesendet: Dienstag, 21. April 2020 10:24
An: IMISE/snik-ontology <snik-ontology@noreply.github.com>
Cc: BirgitSNIK <birgit.schneider@imise.uni-leipzig.de>; Assign <assign@noreply.github.com>
Betreff: Re: [IMISE/snik-ontology] ciox:Krankenhausverwaltung skos:closeMatch bb:HospitalAdministrationDepartment v2018-10-22 (#180)
 
Ich kann Annas Zettel leider nirgendwo finden und hoffe, den haben wir nicht entsorgt als sie gegangen ist. @ThomasPause https://github.com/ThomasPause: Kannst du dich daran erinnern, haben wir vielleicht ein Foto davon?

**Comment by KonradHoeffner** [comment]
@BirgitSNIK: Das war ein schwarzer A3-Ausdruck von einem SNIK Teilgraph, gute Idee mit der Bachelorarbeit aber das war eine Aufgabe außerhalb davon, daher denke ich nicht, dass es dort drin ist.

**Comment by BirgitSNIK** [comment]
Ja, so schwarze Zettel hab ich auch mal gesehen, aber zu 99% liegt bei mir keiner rum – tut mir Leid!
Birgit
 
Von: Konrad Höffner <notifications@github.com> 
Gesendet: Dienstag, 21. April 2020 10:51
An: IMISE/snik-ontology <snik-ontology@noreply.github.com>
Cc: BirgitSNIK <birgit.schneider@imise.uni-leipzig.de>; Mention <mention@noreply.github.com>
Betreff: Re: [IMISE/snik-ontology] ciox:Krankenhausverwaltung skos:closeMatch bb:HospitalAdministrationDepartment v2018-10-22 (#180)
 
@BirgitSNIK https://github.com/BirgitSNIK: Das war ein schwarzer A3-Ausdruck von einem SNIK Teilgraph, gute Idee mit der Bachelorarbeit aber das war eine Aufgabe außerhalb davon, daher denke ich nicht, dass es dort drin ist.
